src/core/expr/fnary/rowminmax.cc

```diff
@@ -56,37 +59,76 @@ static bool op_rowminmax(size_t i, T* out, const colvec& columns) {
   return minmax_valid;
 }
 
-
 template <typename T, bool MIN>
+static bool op_rowargminmax(size_t i, int64_t* out, const colvec& columns) {
+  bool minmax_valid = false;
+  std::pair<T,int64_t> minmax(0, 0);
+  int64_t count = -1;
+  for (const auto& col : columns) {
```

Comment:
If you also need an index, why not just do `for (size_t i = 0; i < columns.size(); ++i)` here?